modules/compliance-applying.adoc

```diff
@@ -16,6 +16,11 @@ After the Compliance Operator processes the applied remediation, the `status.App
 
 Note that when the Machine Config Operator applies a new `MachineConfig` object to nodes in a pool, all the nodes belonging to the pool are rebooted. This might be inconvenient when applying multiple remediations, each of which re-renders the composite `75-$scan-name-$suite-name` `MachineConfig` object. To prevent applying the remediation immediately, you can pause the machine config pool by setting the `.spec.paused` attribute of a `MachineConfigPool` object to `true`.
 
+[NOTE]
+====
+Make sure the pools are unpaused when the CA cert rotation happens. If the MCPs are paused, the MCO cannot push the newly-rotated certs to those nodes. This causes the cluster to become degraded and causes failure in multiple `oc` commands, including but not limited to `oc debug`, `oc logs`, `oc exec`, and `oc attach`. You receive alerts in the Alerting UI of the {product-title} web console if an MCP is paused when the certificates are rotated.
```

Comment:
Technically, "including but not limited to" should have commas to set it off the parenthetical part ("including, but not limited to,"), but that would yield: "multiple `oc` commands, including, but not limited to, `oc debug`, `oc logs`, `oc exec`, and `oc attach`" which imo is pretty clunky. 

Might be simplest/most minimalist to just let "including: do the work as below:

```suggestion
Make sure the pools are unpaused when the CA certificate rotation happens. If the MCPs are paused, the MCO cannot push the newly rotated certificates to those nodes. This causes the cluster to become degraded and causes failure in multiple `oc` commands, including `oc debug`, `oc logs`, `oc exec`, and `oc attach`. You receive alerts in the Alerting UI of the {product-title} web console if an MCP is paused when the certificates are rotated.
```

Either way NB s/`newly-rotated`/`newly rotated` which is easy to miss in the little diff preview :slightly_smiling_face: